frontend/public/models/index.ts

```diff
@@ -3,7 +3,11 @@ import { K8sKind } from '../module/k8s';
 export const PrometheusModel: K8sKind = {
   kind: 'Prometheus',
   label: 'Prometheus',
+  // t('k8s~Prometheus')
+  labelKey: 'k8s~Prometheus',
```

Comment:
I was trying to think of a way to avoid having 4 properties in the model for labels, but I don't have a great suggestion :( It would be good to remove `label` and `labelPlural` from models that have a `labelKey`, but we'd need to sweep the code for everywhere we use `model.label`. That probably needs to be a follow on.

Reply:
I'll add a follow-on task in Jira for this.

Reply:
Added https://issues.redhat.com/browse/CONSOLE-2517.